pkg/virt-api/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -30,8 +30,8 @@ import (
 	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
 )
 
-func ServeVMICreate(resp http.ResponseWriter, req *http.Request, clusterConfig *virtconfig.ClusterConfig) {
-	validating_webhooks.Serve(resp, req, &admitters.VMICreateAdmitter{ClusterConfig: clusterConfig})
+func ServeVMICreate(resp http.ResponseWriter, req *http.Request, clusterConfig *virtconfig.ClusterConfig, validators ...admitters.ValidateSpec) {
```

Comment:
nit: "validators" is a bit too broad IMO:
```suggestion
func ServeVMICreate(resp http.ResponseWriter, req *http.Request, clusterConfig *virtconfig.ClusterConfig, specValidators ...admitters.ValidateSpec) {
```

Reply:
Done